order-api/src/main/java/com/smoothstack/order/configuration/AwsSesConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package com.smoothstack.order.configuration;
+
+import com.amazonaws.auth.EnvironmentVariableCredentialsProvider;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AwsSesConfiguration {
+
+    private final String region;
+
+    public AwsSesConfiguration(@Value("${email.region}") String region) {
+        this.region = region;
+    }
+
+
+    @Bean
+    public AmazonSimpleEmailService amazonSimpleEmailService() {
+        return AmazonSimpleEmailServiceClientBuilder.standard()
+                .withCredentials(new EnvironmentVariableCredentialsProvider())
```

Comment:
We gotta change this to DefaultCredetialChainProvider(). It works with environment variables and IAM roles

Reply:
will do